### ISSUE TITLE
[BUG] reformat timestamp from alerts to be valid RFC3339

### ISSUE THREAD

**Issue by frikilax** [VultureProject/darwin#166]
**Describe the bug**
Current timestamp in darwin alerts is not a valid format:
2020-02-24UTC10:57:31+0000 instead of 2020-02-24T10:57:31+00:00

**Plateform (please complete the following information):**
 - OS (version): all
 - Darwin version: since 1.0

**To Reproduce**
1. tail /var/darwin/alerts.log
2. Generate an alert
3. time should not be a valid RFC3339 format

**Expected behavior**
time field should follow RFC3339 specification (or ISO8601)

**Screenshots/logs**
```json
{"evt_id": "61956d56-3b03-4988-8dc6-a8cf499269f5", "time": "2020-02-24UTC10:57:31+0000", "filter": "content_inspection", "certitude": 100, "yara_match": [{"rule":"eicar","tags":[]}]}
```

**Comment by frikilax** [comment]
potentially linked to #155

**Comment by frikilax** [comment]
resolved by #187
